package.json

```diff
@@ -101,9 +98,9 @@
     "docker-build-ci": "node deploy/docker/create-docker-context.js --build",
     "start": "terriajs-server --config-file serverconfig.json",
     "gulp": "gulp",
-    "postinstall": "echo 'Installation successful. What to do next:\\n  yarn gulp dev   # Starts the server on port 3001, builds TerriaMap and dependencies, and rebuilds if files change.'",
```

Comment:
The wording in the yarn documentation is quite strong to avoid postinstall scripts (https://v3.yarnpkg.com/advanced/lifecycle-scripts#a-note-about-postinstall), and since this is just an echo statement I removed it.